adam/curriculum/phase1_curriculum.py

```diff
@@ -298,6 +303,54 @@ def _make_object_on_object_curriculum() -> _Phase1InstanceGroup:
     )
 
 
+def _make_jump_curriculum():
+    # make an object jump
+    ground = object_variable("ground_0", GROUND)
+    jumper = object_variable("jumper_0", THING, required_properties=[CAN_JUMP])
+    object_0 = object_variable("object_0", THING, required_properties=[CAN_BE_JUMPED])
+
+    # "A person jumps"
+    jump_on_ground = Phase1SituationTemplate(
+        object_variables=[jumper],
+        actions=[
+            Action(
+                JUMP,
+                argument_roles_to_fillers=[(AGENT, jumper)],
+                during=DuringAction(at_some_point=[above(jumper, ground)]),
```

Comment:
This s encoded in the action description of `JUMP` and does not need to be specified here